lambdanotes/src/components/noteform.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+export default class Noteform extends Component {
+    constructor(props){
+      super(props)
+      this.state = {
+        title: '',
+        noteContent: '',
+        
+      }
+    }
+    addNote = () => {
+        const { title, noteContent } = this.state;
+        axios
+          .post('https://fe-notes.herokuapp.com/note/create', {title, noteContent})
+          .then(response => {
+            this.setState({ notes: response.data, title: "", noteContent: "" });
+            window.location.reload();
+          })
+          .catch(err => console.log('note could not be added', err));
+      };
+    
+      inputHandler = e => {
+        this.setState({[e.target.name]: e.target.value})
+      }
+    
+      
+      render(){
+        return (
+          <div>
+            <h2>Create New Note:</h2>
+            <form onSubmit={this.addNote(this.state)}>
```

Comment:
Your addNote function is constantly triggered. If You want to test what's happening add console.log with some text after the post request and see what happens, but i doubt You did not realized Your page is refreshing in constant loop.